schemas/arb.json

```diff
@@ -335,6 +335,10 @@
                 "description": {
                     "type": "string",
                     "description": "A short paragraph describing the resource and how it is being used by the app, and message that need to be passed to localization process and translators."
+                },
+                "comment":{
+                    "type": "string",
+                    "description": "A comment on this resource. This is possibly "
```

Comment:
Message is cropped.  👍 for having a per-entry developer comment (not intended for translators).  This is something I have missed from other localization file formats.  I think it would be useful to point out the intended audience for `comment` in the description (i.e. it is not intended for translators).